net/containernetworking-plugins/Makefile

```diff
@@ -0,0 +1,28 @@
+PORTNAME=	containernetworking-plugins
+DISTVERSION=	0.1
+CATEGORIES=	net
+
+MAINTAINER=	dfr@FreeBSD.org
+COMMENT=	Networking plugins for container networking support
+WWW=		https://github.com/containernetworking/plugins
+
+LICENSE=	GPLv2
+
+USES=		go:no_targets
+BUILD_DEPENDS=	bash:shells/bash
+
+USE_GITHUB=	yes
+GH_ACCOUNT=	dfr
+GH_PROJECT=	plugins
+GH_TAGNAME=	freebsd
+
+do-build:
+	cd ${WRKSRC} && env GO=${GO_CMD} ./build_freebsd.sh
+
+do-install:
+	${MKDIR} ${STAGEDIR}${PREFIX}/libexec/cni
```

Comment:
Don't `${INSTALL_*}` commands create these dirs if they don't exist?

Reply:
No they don't - I get errors like this:
```
cd /wrkdirs/usr/ports/net/containernetworking-plugins/work/plugins-freebsd && install  -s -m 555 -m 755 bin/* /wrkdirs/usr/ports/net/containernetworking-plugins/work/stage/usr/local/libexec/cni
install: target directory `/wrkdirs/usr/ports/net/containernetworking-plugins/work/stage/usr/local/libexec/cni' does not exist
```
Not sure what the right way is for ports - I would prefer to use 'install -d' instead of 'mkdir -p' but I don't see a matching INSTALL_* setting in bsd.port.mk

Reply:
Grepping through various ports in the tree reveals that `${MKDIR}` + `${INSTALL_PROGRAM}` is indeed way to go.